SIL.DictionaryServices.Tests/Lift/LiftWriterTests.cs

```diff
@@ -902,13 +902,12 @@ public void EntryHasDateModified()
 		}
 
 		/// <summary>
-		/// Regression: WS-34576
+		/// Regression: WS-34576, LT-20698
 		/// </summary>
 		[Test]
 		public void Add_CultureUsesPeriodForTimeSeparator_DateAttributesOutputWithColon()
 		{
-			var culture = new CultureInfo("en-US");
-			culture.DateTimeFormat.TimeSeparator = ".";
+			var culture = new CultureInfo("de-DE") {DateTimeFormat = {TimeSeparator = "."}};
```

Comment:
Might not be strictly necessary, but maybe it would be good to make two test cases: one for en-US and one for de-DE. At least that would kind of preserve the history of this test's original behavior.

Reply:
done